docs/src/main/sphinx/sql/show-create-function.md

```diff
@@ -2,20 +2,60 @@
 
 ## Synopsis
 
-```text
+```sql
 SHOW CREATE FUNCTION function_name
 ```
 
 ## Description
 
-Show the SQL statement that creates the specified function.
+Shows the SQL statement that created the specified function.
```

Comment:
Dont change this .. since all other docs are like that .. we should stay consistent.

Reply:
Need to undo